resources/components/ImageCard.vue

```diff
@@ -19,17 +19,37 @@
 				</ul>
 			</div>
 
-			<div class="wbmad-buttons">
-				BUTTONS GO HERE
+			<div class="wbmad-action-buttons">
+				<base-button
```

Comment:
Documentation note: it took me way too long to figure out that Pascal and camel casing isn't allowed in tag names (I'm used to Pascal case for components in React) and that attributes/props have to be all lowercase (is this right? I couldn't use onClick as an attribute...)